CONTRIBUTING.md

```diff
@@ -1,7 +1,7 @@
 # Contributing
 
 Contributions are always welcome. Before contributing please read the
-[code of conduct](./CODE_OF_CONDUCT.md) and [search the issue tracker](../../issues); your issue may have already been discussed or fixed in `master`. To contribute,
+[code of conduct](https://github.com/newrelic/.github/blob/main/CODE_OF_CONDUCT.md) and [search the issue tracker](../../issues); your issue may have already been discussed or fixed in `master`. To contribute,
```

Comment:
So, the file this links to is just a small file with a link to the real CoC at https://opensource.newrelic.com/code-of-conduct/.  I think we should just link straight to that URL unless we know of a reason not to.

Reply:
The linter suggested pointing to this intermediate file.  Who do we need to talk to fix that so that it doesn't throw another error?